### DISCUSSION TITLE
[Help] Regarding unicode support and usability

### DISCUSSION THREAD

**Discussion by itsnotsaved** [algolia/instantsearch#4810] [Q&A] [answered]
1. Where can i find more details or documentations about 'algolia unicode support'?
2. How many languages currently supported? and about if there any bugs related them
3. Does algolia search currently support for languages similar dhivehi (dv) , sinhala (si)?
  (ISO 639-1)

_I'm just asking this question from random repository however i believe developers capable to answer for that otherwise point me where should i ask this question and get response soon_

**Comment by Haroenv** [comment, marked as answer]
As you already mentioned, this isn't really the right place to ask, as it's related to the backend, support@algolia.com is a much more relevant place.

For the languages Dhivehi and Sinhala I'm not sure, as they are more intricate languages with regard to prefixes and suffixes. I tried out a sentence like 'ඒක අලුත් ද කියලා මම දන්නේ නැහැ' (which I got from Wikipedia, sorry, I don't know what it means', and saw that the matching actually happens on sub characters, "නැහ" as query will highlight a match as " නැහ/break/ැ".

I can tell from the setting that we don't have any particular handling of both Sinhala and Dhivehi: https://www.algolia.com/doc/api-reference/api-parameters/queryLanguages/

For more info, please contact support, and know that I just got this info by trying it out, not by looking at definite data.

**Comment by itsnotsaved** [comment]
Thank you for the reply. 'ඒක අලුත් ද කියලා මම දන්නේ නැහැ' it exactly means 'I don't know if it is new' 

>and saw that the matching actually happens on sub characters, "නැහ" as query will highlight a match as " නැහ/break/ැ".

it should be remain as 'නැහැ' without breaks. it's similar to Je voudrais un café. --> cafe/brake/ ´

>it's related to the backend, support@algolia.com is a much more relevant place.

Sure 👍😊
